app/views/catalog/_osd_or_embed_default.html.erb

```diff
@@ -1,5 +1,8 @@
+<% @viewer = Viewer.find_by(exhibit: @exhibit) %>
```

Comment:
Can we make this a local variable?  If it's needed by any other partials, we should pass it as a local.